### PR TITLE
added the ability to sort by fields by default or in a defined way.

### PR DESCRIPTION
**Pull request by opbro**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 dist/
 pg_es_fdw.egg-info/
 .make
+.vscode/
```
Comment (matthewfranglen) [inline .gitignore:4]: This is a change specific to your editor that should be handled using a global gitignore: https://stackoverflow.com/questions/7335420/global-git-ignore

File: pg_es_fdw/__init__.py [hunk 2]
```diff
@@ -214,8 +218,22 @@ def _get_query(self, quals):
...
+        
+        return next(
+            (
+                qualifier.value
+                for qualifier in quals
+                if qualifier.field_name == self.sort_column and qualifier.value
+            ),
+            self.default_sort
+        )
+        return self.default_sort
```
Comment (matthewfranglen) [inline pg_es_fdw/__init__.py:234]: Redundant return statement

File: README.md [hunk 3]
```diff
@@ -197,6 +197,82 @@ WHERE
...
+ 7    | 2020-09-25 09:06:49.433148 | unit:asc
+ 8    | 2020-09-25 09:06:50.445831 | unit:asc
+ 9    | 2020-09-25 09:06:51.457017 | unit:asc
+```
+
+
+
+
+
 This uses the [URI Search](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-uri-request.html) from Elastic Search.
```
Comment (matthewfranglen) [inline README.md:276]: I appreciate you documenting the new functionality however you have put it in the middle of this section. Please add it after the line I have commented on. It would also be great to have a single blank line between sections instead of 5.

File: pg_es_fdw/__init__.py [hunk 4]
```diff
@@ -227,14 +245,18 @@ def _convert_response_row(self, row_data, columns, query):
...
+        return dict(
+            [
+                (column, self._convert_response_column(column, row_data))
+                for column in columns
+                if column in row_data["_source"]
+                or column == self.rowid_column
+                or column == self.score_column
+            ]
+            + [(self.sort_column, sort)]
+        )
```
Comment (matthewfranglen) [inline pg_es_fdw/__init__.py:259]: It's easier to read this if you assign the original dict comprehension to a variable and then assign the sort_column key.

File: setup.py [hunk 5]
```diff
@@ -12,7 +12,7 @@
     setup(
         name="pg_es_fdw",
         packages=["pg_es_fdw"],
-        version="0.8.0",
+        version="0.9.0",
```
Comment (matthewfranglen) [inline setup.py:15]: I use git flow for managing releases so the version bump will be in a separate commit.

**Review by matthewfranglen — COMMENTED** [review]
Thank you for submitting this PR. I've added a few comments where things could be improved.

I am happy to pick this work up at this point if you wish, or you can address the comments and then I will review again.

File: README.md [hunk 6]
```diff
@@ -197,6 +197,82 @@ WHERE
 ;
 ```
 
+#### Using the default sort and sort column
+When you want to query and sort before returning results.
+
+### Creating the Table
```
Comment (matthewfranglen) [inline README.md:203]: This should have 5 hashes as it is a subsection of the sort column section

File: .gitignore (hunk 1 above)
Comment (opbro) [reply .gitignore:4]: as 

> Thank you for submitting this PR. I've added a few comments where things could be improved.
> 
> I am happy to pick this work up at this point if you wish, or you can address the comments and then I will review again.

I can work on this and resubmit. Thanks for reviewing it.

**Comment by matthewfranglen** [comment]
Awesome. I'm going to run the tests and then release this.

**Comment by matthewfranglen** [comment]
I added a test to validate that sorting doesn't break the existing functionality. I've released this as version 0.9.0.

Thank you very much for submitting this PR. :medal_sports:
